package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "marvin",
+  "version": "0.0.0",
+  "description": "An API driven skin for MediaWiki",
+  "scripts": {
+    "start": "run-p -n dev:*",
+    "dev:client": "NODE_ENV=development webpack -w",
+    "dev:server": "NODE_ENV=development nodemon -w dist/client -w src/server/ src/server/index.js",
+    "prod:build": "NODE_ENV=production webpack -p",
+    "format": "prettier --write 'src/**/*.js'",
+    "lint": "eslint 'src/**/*.js'",
```

Comment:
Thanks for the node-glob tip!

> Regarding --ext .js,.json,.html it seems overkill to set up linting for them given we don't have any files of that kind, feel free to submit a followup commit, I'd be happy to merge.
We do have package.json but I suppose we won't be able to run any scripts if that's broken. There's some other feedback on eslint-plugin-json so let's punt!

> Re: Replace 'src/**/*.js', I believe that having the explicit path to the JS files there kind of makes it easy to check the paths of the files for other tools like prettier. Happy to re-evaluate later if we increase the amount of eslint config
I don't mind punting on this too. My thoughts were that I prefer linting to default to complaining rather abstaining. I might try to make a more compelling argument in a future PR.